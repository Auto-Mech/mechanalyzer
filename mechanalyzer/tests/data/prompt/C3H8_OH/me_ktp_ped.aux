W0:
1-th ROTOR:
first point frequency estimate = 66.3633 1/cm
harmonic frequency = 120 1/cm
harmonic  frequency at first point = 48 1/cm
fundamental frequency = 42 1/cm
2-th ROTOR:
first point frequency estimate = 15.8684 1/cm
harmonic frequency = 29 1/cm
harmonic  frequency at first point = 12 1/cm
fundamental frequency = 8 1/cm
3-th ROTOR:
first point frequency estimate = 222.161 1/cm
harmonic frequency = 225 1/cm
harmonic  frequency at first point = 225 1/cm
fundamental frequency = 212 1/cm
C3H8:
1-th ROTOR:
first point frequency estimate = 234.238 1/cm
harmonic frequency = 241 1/cm
harmonic  frequency at first point = 241 1/cm
fundamental frequency = 220 1/cm
2-th ROTOR:
first point frequency estimate = 234.238 1/cm
harmonic frequency = 241 1/cm
harmonic  frequency at first point = 241 1/cm
fundamental frequency = 220 1/cm
OH:
CH3CH2CH2:
1-th ROTOR:
first point frequency estimate = 98.8802 1/cm
harmonic frequency = 89 1/cm
harmonic  frequency at first point = 107 1/cm
fundamental frequency = 73 1/cm
2-th ROTOR:
first point frequency estimate = 199.725 1/cm
harmonic frequency = 193 1/cm
harmonic  frequency at first point = 191 1/cm
fundamental frequency = 205 1/cm
H2O:
CH3CHCH3:
1-th ROTOR:
first point frequency estimate = 95.6873 1/cm
harmonic frequency = 127 1/cm
harmonic  frequency at first point = 115 1/cm
fundamental frequency = 80 1/cm
2-th ROTOR:
first point frequency estimate = 95.6873 1/cm
harmonic frequency = 127 1/cm
harmonic  frequency at first point = 115 1/cm
fundamental frequency = 80 1/cm
H2O:
B0:
B1:
1-th ROTOR:
first point frequency estimate = 66.3633 1/cm
harmonic frequency = 120 1/cm
harmonic  frequency at first point = 48 1/cm
fundamental frequency = 42 1/cm
2-th ROTOR:
first point frequency estimate = 15.8684 1/cm
harmonic frequency = 29 1/cm
harmonic  frequency at first point = 12 1/cm
fundamental frequency = 8 1/cm
3-th ROTOR:
first point frequency estimate = 222.161 1/cm
harmonic frequency = 225 1/cm
harmonic  frequency at first point = 225 1/cm
fundamental frequency = 212 1/cm
B2:
1-th ROTOR:
first point frequency estimate = 65.9447 1/cm
harmonic frequency = 119 1/cm
harmonic  frequency at first point = 48 1/cm
fundamental frequency = 42 1/cm
2-th ROTOR:
first point frequency estimate = 222.524 1/cm
harmonic frequency = 225 1/cm
harmonic  frequency at first point = 225 1/cm
fundamental frequency = 212 1/cm
3-th ROTOR:
first point frequency estimate = 222.524 1/cm
harmonic frequency = 225 1/cm
harmonic  frequency at first point = 225 1/cm
fundamental frequency = 212 1/cm
Pressure = 1.01324 bar	 Temperature = 400 K
Pressure = 1.01324 bar	 Temperature = 600 K
Pressure = 1.01324 bar	 Temperature = 800 K
Pressure = 1.01324 bar	 Temperature = 1200 K
Pressure = 1.01324 bar	 Temperature = 1800 K
Pressure = 1.01324 bar	 Temperature = 2000 K
