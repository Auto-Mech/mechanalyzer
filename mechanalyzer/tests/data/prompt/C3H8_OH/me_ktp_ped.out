Wells (G - ground energy, D - dissociation limit, kcal/mol):
       W         G         D
      W0        -6      -2.2

Bimolecular Products (G - ground energy, kcal/mol):
                  P         G
            C3H8+OH      -2.2
      CH3CH2CH2+H2O     -20.1
       CH3CHCH3+H2O     -23.2

Well-to-Bimolecular Barriers (H/G - barrier height/well depth, kcal/mol)
       B         H  W         G             P
      B0      -2.2 W0        -6       C3H8+OH
      B1         0 W0        -6 CH3CH2CH2+H2O
      B2     -0.89 W0        -6  CH3CHCH3+H2O

Bound Groups: dissociation barrier (name/ground energy, kcal/mol): wells (name/ground energy, kcal/mol):
     C0/-2.2:  W0/-6

________________________________________________________________________________________

Well Names Translation:
 W0  W0
End
Bimolecular Names Translation:
 P0  C3H8+OH
 P1  CH3CH2CH2+H2O
 P2  CH3CHCH3+H2O
End
Unimolecular Rate Units: 1/sec;  Bimolecular Rate Units: cm^3/sec

______________________________________________________________________________________

Species-Species Rate Tables:

Temperature = 400 K

High Pressure Rate Coefficients:

From\To         W0        P0        P1        P2
W0             ***  3.53e+11  1.47e+09  5.38e+08
P0        1.26e-10       ***       ***       ***
P1        6.42e-24       ***       ***       ***
P2        6.31e-26       ***       ***       ***

Temperature = 400 K    Pressure = 1 atm

From\To         W0        P0        P1        P2
W0             ***       ***       ***       ***
P0             ***  7.03e-13  5.14e-13  1.89e-13
P1             ***  6.31e-24  6.32e-24  1.62e-26
P2             ***  6.23e-26  4.34e-28  6.27e-26

Temperature = 600 K

High Pressure Rate Coefficients:

From\To         W0        P0        P1        P2
W0             ***  1.36e+12  1.81e+10  5.25e+09
P0        1.15e-10       ***       ***       ***
P1        3.02e-20       ***       ***       ***
P2        9.25e-22       ***       ***       ***

Temperature = 600 K    Pressure = 1 atm

From\To         W0        P0        P1        P2
W0             ***       ***       ***       ***
P0             ***  1.93e-12  1.49e-12  4.34e-13
P1             ***  2.93e-20  2.95e-20  1.56e-22
P2             ***  9.02e-22  1.65e-23  9.18e-22

Temperature = 800 K

High Pressure Rate Coefficients:

From\To         W0        P0        P1        P2
W0             ***  2.33e+12  6.43e+10  1.68e+10
P0        1.12e-10       ***       ***       ***
P1        2.45e-18       ***       ***       ***
P2         1.4e-19       ***       ***       ***

Temperature = 800 K    Pressure = 1 atm

From\To         W0        P0        P1        P2
W0             ***       ***       ***       ***
P0             ***  3.72e-12  2.95e-12  7.72e-13
P1             ***  2.33e-18  2.35e-18  2.11e-20
P2             ***  1.34e-19  4.63e-21  1.39e-19

Temperature = 1.2e+03 K

High Pressure Rate Coefficients:

From\To         W0        P0        P1        P2
W0             ***  3.33e+12   2.3e+11  5.41e+10
P0        1.17e-10       ***       ***       ***
P1        2.68e-16       ***       ***       ***
P2        2.98e-17       ***       ***       ***

Temperature = 1.2e+03 K    Pressure = 1 atm

From\To         W0        P0        P1        P2
W0             ***       ***       ***       ***
P0             ***  9.06e-12  7.33e-12  1.73e-12
P1             ***  2.43e-16  2.48e-16  4.56e-18
P2             ***  2.71e-17  2.16e-18  2.93e-17

Temperature = 1.8e+03 K

High Pressure Rate Coefficients:

From\To         W0        P0        P1        P2
W0             ***   3.5e+12  5.39e+11  1.18e+11
P0        1.35e-10       ***       ***       ***
P1           9e-15       ***       ***       ***
P2        1.59e-15       ***       ***       ***

Temperature = 1.8e+03 K    Pressure = 1 atm

From\To         W0        P0        P1        P2
W0             ***       ***       ***       ***
P0             ***   2.1e-11  1.72e-11  3.79e-12
P1             ***  7.43e-15   7.7e-15  2.75e-16
P2             ***  1.32e-15  2.22e-16  1.54e-15

Temperature = 2e+03 K

High Pressure Rate Coefficients:

From\To         W0        P0        P1        P2
W0             ***  3.43e+12   6.4e+11  1.38e+11
P0        1.42e-10       ***       ***       ***
P1        1.95e-14       ***       ***       ***
P2        3.77e-15       ***       ***       ***

Temperature = 2e+03 K    Pressure = 1 atm

From\To         W0        P0        P1        P2
W0             ***       ***       ***       ***
P0             ***  2.59e-11  2.13e-11  4.61e-12
P1             ***  1.56e-14  1.62e-14  6.81e-16
P2             ***  3.03e-15  6.11e-16  3.64e-15

______________________________________________________________________________________

High Pressure Rate Coefficients (Temperature-Species Rate Tables):

Reactant = W0
   T(K)        W0        P0        P1        P2
    400       ***  3.53e+11  1.47e+09  5.38e+08
    600       ***  1.36e+12  1.81e+10  5.25e+09
    800       ***  2.33e+12  6.43e+10  1.68e+10
1.2e+03       ***  3.33e+12   2.3e+11  5.41e+10
1.8e+03       ***   3.5e+12  5.39e+11  1.18e+11
  2e+03       ***  3.43e+12   6.4e+11  1.38e+11

Reactant = P0
   T(K)        W0        P0        P1        P2
    400  1.26e-10       ***       ***       ***
    600  1.15e-10       ***       ***       ***
    800  1.12e-10       ***       ***       ***
1.2e+03  1.17e-10       ***       ***       ***
1.8e+03  1.35e-10       ***       ***       ***
  2e+03  1.42e-10       ***       ***       ***

Reactant = P1
   T(K)        W0        P0        P1        P2
    400  6.42e-24       ***       ***       ***
    600  3.02e-20       ***       ***       ***
    800  2.45e-18       ***       ***       ***
1.2e+03  2.68e-16       ***       ***       ***
1.8e+03     9e-15       ***       ***       ***
  2e+03  1.95e-14       ***       ***       ***

Reactant = P2
   T(K)        W0        P0        P1        P2
    400  6.31e-26       ***       ***       ***
    600  9.25e-22       ***       ***       ***
    800   1.4e-19       ***       ***       ***
1.2e+03  2.98e-17       ***       ***       ***
1.8e+03  1.59e-15       ***       ***       ***
  2e+03  3.77e-15       ***       ***       ***

Capture/Escape Rate Coefficients:

   T(K)        W0        P0        P1        P2
    400  3.55e+11  1.26e-10  6.42e-24  6.31e-26
    600  1.38e+12  1.15e-10  3.02e-20  9.25e-22
    800  2.41e+12  1.12e-10  2.45e-18   1.4e-19
1.2e+03  3.62e+12  1.17e-10  2.68e-16  2.98e-17
1.8e+03  4.16e+12  1.35e-10     9e-15  1.59e-15
  2e+03  4.21e+12  1.42e-10  1.95e-14  3.77e-15

______________________________________________________________________________________

Pressure-Species Rate Tables:

Reactant = W0   Temperature = 400 K

   P(atm)        P0        P1        P2      Loss
        1       ***       ***       ***       ***
      O-O  3.53e+11  1.47e+09  5.38e+08  3.55e+11

Reactant = W0   Temperature = 600 K

   P(atm)        P0        P1        P2      Loss
        1       ***       ***       ***       ***
      O-O  1.36e+12  1.81e+10  5.25e+09  1.38e+12

Reactant = W0   Temperature = 800 K

   P(atm)        P0        P1        P2      Loss
        1       ***       ***       ***       ***
      O-O  2.33e+12  6.43e+10  1.68e+10  2.41e+12

Reactant = W0   Temperature = 1.2e+03 K

   P(atm)        P0        P1        P2      Loss
        1       ***       ***       ***       ***
      O-O  3.33e+12   2.3e+11  5.41e+10  3.62e+12

Reactant = W0   Temperature = 1.8e+03 K

   P(atm)        P0        P1        P2      Loss
        1       ***       ***       ***       ***
      O-O   3.5e+12  5.39e+11  1.18e+11  4.16e+12

Reactant = W0   Temperature = 2e+03 K

   P(atm)        P0        P1        P2      Loss
        1       ***       ***       ***       ***
      O-O  3.43e+12   6.4e+11  1.38e+11  4.21e+12

Reactant = P0   Temperature = 400 K

   P(atm)        W0        P1        P2      Loss
        1       ***  5.14e-13  1.89e-13  7.03e-13
      O-O  1.26e-10       ***       ***  1.26e-10

Reactant = P0   Temperature = 600 K

   P(atm)        W0        P1        P2      Loss
        1       ***  1.49e-12  4.34e-13  1.93e-12
      O-O  1.15e-10       ***       ***  1.15e-10

Reactant = P0   Temperature = 800 K

   P(atm)        W0        P1        P2      Loss
        1       ***  2.95e-12  7.72e-13  3.72e-12
      O-O  1.12e-10       ***       ***  1.12e-10

Reactant = P0   Temperature = 1.2e+03 K

   P(atm)        W0        P1        P2      Loss
        1       ***  7.33e-12  1.73e-12  9.06e-12
      O-O  1.17e-10       ***       ***  1.17e-10

Reactant = P0   Temperature = 1.8e+03 K

   P(atm)        W0        P1        P2      Loss
        1       ***  1.72e-11  3.79e-12   2.1e-11
      O-O  1.35e-10       ***       ***  1.35e-10

Reactant = P0   Temperature = 2e+03 K

   P(atm)        W0        P1        P2      Loss
        1       ***  2.13e-11  4.61e-12  2.59e-11
      O-O  1.42e-10       ***       ***  1.42e-10

Reactant = P1   Temperature = 400 K

   P(atm)        W0        P0        P2      Loss
        1       ***  6.31e-24  1.62e-26  6.32e-24
      O-O  6.42e-24       ***       ***  6.42e-24

Reactant = P1   Temperature = 600 K

   P(atm)        W0        P0        P2      Loss
        1       ***  2.93e-20  1.56e-22  2.95e-20
      O-O  3.02e-20       ***       ***  3.02e-20

Reactant = P1   Temperature = 800 K

   P(atm)        W0        P0        P2      Loss
        1       ***  2.33e-18  2.11e-20  2.35e-18
      O-O  2.45e-18       ***       ***  2.45e-18

Reactant = P1   Temperature = 1.2e+03 K

   P(atm)        W0        P0        P2      Loss
        1       ***  2.43e-16  4.56e-18  2.48e-16
      O-O  2.68e-16       ***       ***  2.68e-16

Reactant = P1   Temperature = 1.8e+03 K

   P(atm)        W0        P0        P2      Loss
        1       ***  7.43e-15  2.75e-16   7.7e-15
      O-O     9e-15       ***       ***     9e-15

Reactant = P1   Temperature = 2e+03 K

   P(atm)        W0        P0        P2      Loss
        1       ***  1.56e-14  6.81e-16  1.62e-14
      O-O  1.95e-14       ***       ***  1.95e-14

Reactant = P2   Temperature = 400 K

   P(atm)        W0        P0        P1      Loss
        1       ***  6.23e-26  4.34e-28  6.27e-26
      O-O  6.31e-26       ***       ***  6.31e-26

Reactant = P2   Temperature = 600 K

   P(atm)        W0        P0        P1      Loss
        1       ***  9.02e-22  1.65e-23  9.18e-22
      O-O  9.25e-22       ***       ***  9.25e-22

Reactant = P2   Temperature = 800 K

   P(atm)        W0        P0        P1      Loss
        1       ***  1.34e-19  4.63e-21  1.39e-19
      O-O   1.4e-19       ***       ***   1.4e-19

Reactant = P2   Temperature = 1.2e+03 K

   P(atm)        W0        P0        P1      Loss
        1       ***  2.71e-17  2.16e-18  2.93e-17
      O-O  2.98e-17       ***       ***  2.98e-17

Reactant = P2   Temperature = 1.8e+03 K

   P(atm)        W0        P0        P1      Loss
        1       ***  1.32e-15  2.22e-16  1.54e-15
      O-O  1.59e-15       ***       ***  1.59e-15

Reactant = P2   Temperature = 2e+03 K

   P(atm)        W0        P0        P1      Loss
        1       ***  3.03e-15  6.11e-16  3.64e-15
      O-O  3.77e-15       ***       ***  3.77e-15

______________________________________________________________________________________

Temperature-Species Rate Tables:

Reactant = W0   Pressure = 1 atm

   T(K)        P0        P1        P2      Loss   Capture
    400       ***       ***       ***       ***  3.55e+11
    600       ***       ***       ***       ***  1.38e+12
    800       ***       ***       ***       ***  2.41e+12
1.2e+03       ***       ***       ***       ***  3.62e+12
1.8e+03       ***       ***       ***       ***  4.16e+12
  2e+03       ***       ***       ***       ***  4.21e+12

Reactant = P0   Pressure = 1 atm

   T(K)        W0        P1        P2      Loss   Capture
    400       ***  5.14e-13  1.89e-13  7.03e-13  1.26e-10
    600       ***  1.49e-12  4.34e-13  1.93e-12  1.15e-10
    800       ***  2.95e-12  7.72e-13  3.72e-12  1.12e-10
1.2e+03       ***  7.33e-12  1.73e-12  9.06e-12  1.17e-10
1.8e+03       ***  1.72e-11  3.79e-12   2.1e-11  1.35e-10
  2e+03       ***  2.13e-11  4.61e-12  2.59e-11  1.42e-10

Reactant = P1   Pressure = 1 atm

   T(K)        W0        P0        P2      Loss   Capture
    400       ***  6.31e-24  1.62e-26  6.32e-24  6.42e-24
    600       ***  2.93e-20  1.56e-22  2.95e-20  3.02e-20
    800       ***  2.33e-18  2.11e-20  2.35e-18  2.45e-18
1.2e+03       ***  2.43e-16  4.56e-18  2.48e-16  2.68e-16
1.8e+03       ***  7.43e-15  2.75e-16   7.7e-15     9e-15
  2e+03       ***  1.56e-14  6.81e-16  1.62e-14  1.95e-14

Reactant = P2   Pressure = 1 atm

   T(K)        W0        P0        P1      Loss   Capture
    400       ***  6.23e-26  4.34e-28  6.27e-26  6.31e-26
    600       ***  9.02e-22  1.65e-23  9.18e-22  9.25e-22
    800       ***  1.34e-19  4.63e-21  1.39e-19   1.4e-19
1.2e+03       ***  2.71e-17  2.16e-18  2.93e-17  2.98e-17
1.8e+03       ***  1.32e-15  2.22e-16  1.54e-15  1.59e-15
  2e+03       ***  3.03e-15  6.11e-16  3.64e-15  3.77e-15

______________________________________________________________________________________

Temperature-Pressure Rate Tables:

W0->P0

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1             ***       ***       ***       ***       ***       ***
O-O      3.53e+11  1.36e+12  2.33e+12  3.33e+12   3.5e+12  3.43e+12

W0->P1

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1             ***       ***       ***       ***       ***       ***
O-O      1.47e+09  1.81e+10  6.43e+10   2.3e+11  5.39e+11   6.4e+11

W0->P2

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1             ***       ***       ***       ***       ***       ***
O-O      5.38e+08  5.25e+09  1.68e+10  5.41e+10  1.18e+11  1.38e+11

P0->W0

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1             ***       ***       ***       ***       ***       ***
O-O      1.26e-10  1.15e-10  1.12e-10  1.17e-10  1.35e-10  1.42e-10

P0->P1

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        5.14e-13  1.49e-12  2.95e-12  7.33e-12  1.72e-11  2.13e-11
O-O           ***       ***       ***       ***       ***       ***

P0->P2

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        1.89e-13  4.34e-13  7.72e-13  1.73e-12  3.79e-12  4.61e-12
O-O           ***       ***       ***       ***       ***       ***

P1->W0

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1             ***       ***       ***       ***       ***       ***
O-O      6.42e-24  3.02e-20  2.45e-18  2.68e-16     9e-15  1.95e-14

P1->P0

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        6.31e-24  2.93e-20  2.33e-18  2.43e-16  7.43e-15  1.56e-14
O-O           ***       ***       ***       ***       ***       ***

P1->P2

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        1.62e-26  1.56e-22  2.11e-20  4.56e-18  2.75e-16  6.81e-16
O-O           ***       ***       ***       ***       ***       ***

P2->W0

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1             ***       ***       ***       ***       ***       ***
O-O      6.31e-26  9.25e-22   1.4e-19  2.98e-17  1.59e-15  3.77e-15

P2->P0

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        6.23e-26  9.02e-22  1.34e-19  2.71e-17  1.32e-15  3.03e-15
O-O           ***       ***       ***       ***       ***       ***

P2->P1

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        4.34e-28  1.65e-23  4.63e-21  2.16e-18  2.22e-16  6.11e-16
O-O           ***       ***       ***       ***       ***       ***

W0-> 

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1             ***       ***       ***       ***       ***       ***
O-O      3.55e+11  1.38e+12  2.41e+12  3.62e+12  4.16e+12  4.21e+12

P0-> 

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        7.03e-13  1.93e-12  3.72e-12  9.06e-12   2.1e-11  2.59e-11
O-O      1.26e-10  1.15e-10  1.12e-10  1.17e-10  1.35e-10  1.42e-10

P1-> 

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        6.32e-24  2.95e-20  2.35e-18  2.48e-16   7.7e-15  1.62e-14
O-O      6.42e-24  3.02e-20  2.45e-18  2.68e-16     9e-15  1.95e-14

P2-> 

P\T           400       600       800   1.2e+03   1.8e+03     2e+03
1        6.27e-26  9.18e-22  1.39e-19  2.93e-17  1.54e-15  3.64e-15
O-O      6.31e-26  9.25e-22   1.4e-19  2.98e-17  1.59e-15  3.77e-15

